sympy/concrete/summations.py

```diff
@@ -496,12 +513,6 @@ def is_convergent(self):
         except NotImplementedError:
             pass
 
-        ### ------------- alternating series test ----------- ###
-        dict_val = sequence_term.match((-1)**(sym + p)*q)
-        if not dict_val[p].has(sym) and is_decreasing(dict_val[q], interval):
-            return S.true
```

Comment:
It looks like this and everything below is untested. Are these tests (alternating series etc) not already implemented somewhere else?

Reply:
I don't fully understand, there are already tests for Alternating Series and for everything else that follows. I believe that the issue might be that my p-series limit comparison is just returning a solution, blocking access to some of the functions later in the code. But now the codecov score looks pretty decent, right? And if not, what is it that I fix, I have no clue about codecov?